src/doublylinkedlist.js

```diff
@@ -0,0 +1,158 @@
+'use strict'
+
+class Node {
+
+  constructor( data ) {
+    this.data = data
+    this.next = null
+    this.prev = null
+  }
+
+}
+export default class DoublyLinkedList {
+
+  constructor () {
+      this.head = null
+      this.tail = null
+      this.count = 0
+
+
+  }
+  // Inserts a node (with the provided data) to the head of the list
+  insertFirst ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if head is defined
+    if ( this.head ) {
+      //set old head to next of new node
+      node.next = this.head
+      this.head.prev = node
+      //if head is defined, set new node to head
+    } else {
+      //if not defined set current node to head
+      this.tail = node
+    }
+    this.head = node
+    this.count++
+  }
+  // Returns the size of the list (number of nodes)
+  size () {
+    return this.count
+  }
+
+  // Inserts a node (with the provided data) to the tail of the list
+  insert ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if tail is defined
+    if ( this.head === null ) {
+    //set oldtail next to new node
+      this.head = node
+      this.tail = node
+
+    //set newnode to tail
+    } else {
+      this.tail.next = node
+      node.prev = this.tail
+      this.tail = node
+    }
+    //set new tail to null
+    //this.tail = null
+    this.count++
+  }
+  //Returns the first node containing the provided data,
+  // or -1 if not found
+  find( value ) {
+    let currentNode = this.head
+    while (currentNode !== this.tail) {
+      if (currentNode.data === value) {
+        return currentNode
+      } else {
+        currentNode = currentNode.next
+      }
+    }
+    return currentNode.data === value ? currentNode : -1
+  }
+  // Determines if the list is empty or not
+  isEmpty() {
+    return this.head ? false : true
```

Comment:
This logic is wrong. This is saying if there is a head return false else return true.
So we should just reverse the logic